docs/concepts/Dialog-Command.md

```diff
@@ -0,0 +1,78 @@
+# Dialog Command
+
+`askx dialog` opens a interactive mode, similar to Node's REPL, in which developers can simulate a multi-turn conversation with  Alexa. For each utterance, Dialog command leverages [simulate-skill](https://developer.amazon.com/en-US/docs/alexa/smapi/ask-cli-command-reference.html#simulate-command) to asynchronously poll simulation result until it's available and shows Alexa text responses. **Note:** Skill must have been enabled for simulation before running the dialog command.
+
+Structure of the dialog command is:
+
+`askx dialog [-s|--skill-id <skill-id>] [-l|--locale <locale>] [-g|--stage <stage>] [-r|--replay <file-path>] [-p|--profile <profile>] [--debug]`
+
+Default value for `--profile`is `default`and `--stage` is `development`. Specifying `--debug` appends a debug message to the standard error.
+
+##Dialog command supports two modes:
+
+### INTERACTIVE MODE
+
+This mode allows developers to simulate a multi-turn conversation with Alexa. If `skill-id` is not specified, the command must be run from the skill project root directory. If the environment variable with key `ASK_DEFAULT_DEVICE_LOCALE` is not specified, then locale must be provided as part of arguments. Alternatively, User has an option to record the list of utterances so far in a JSON file, by using a special command `.record`, which will be used in replay mode. User may either keep interacting with the skill or exit the Interactive mode by using a special command, `.quit`.
```

Comment:
* typo: must be `running` from the skill project root directory
* special command I also suggest we list them all as a separate bullet point after the provided UX example, as it may grow into more commands
* for the .quit part, we also need to let users know will the stop of the skill session quit the dialog

Reply:
did not get your last point. Can you elaborate?